ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -319,6 +325,37 @@ public static class Utilities {
 			}
 		}
 
+		internal static bool IsWeakPassword(string password, params string[] additionallyForbiddenWords) {
```

Comment:
Make `additionallyForbiddenWords` `ISet<string>`, it's more generic and gives clearer intention.